### DISCUSSION TITLE
2024 Akash Hackathons Proposal

### DISCUSSION THREAD

**Discussion by CaptainSplince** [akash-network/community#415] [Governance Proposals]
# 2024 Akash Hackathons Proposal

## Overview

Hosting hackathons for the Akash Network has become increasingly imperative, driven by the network's growing prominence. This proposal introduces the idea of hosting 3 Akash Network hackathons throughout 2024.

### Why host Akash Hackathons in 2024?

- **Innovation and Ideation:** By bringing together diverse teams of developers, designers, and entrepreneurs, these events stimulate the creation of novel solutions and applications that leverage the unique features of the Akash Network.

- **Ecosystem Growth:** Encouraging developers to build on the Akash Network expands the range of applications available, attracting more users and stakeholders to the network.

- **Developer Engagement:** Offering hands-on experiences with the Akash platform encourages developers to explore its capabilities, experiment, and contribute to the community.

- **Community Building:** By facilitating collaboration and networking among participants, an Akash hackathon can strengthen the bonds within our community, creating a supportive environment for ongoing development and collaboration.

- **Akash Education:** Hackathons act as educational platforms, providing participants with the opportunity to learn more about the Akash Network. This increased awareness contributes to a more informed and skilled developer community.

Hosting hackathons for the Akash Network in 2024 can be an essential tool for driving innovation, expanding the ecosystem, engaging developers, solving real-world challenges, building a strong community, and accelerating the overall development of the Akash Network. These events serve as a catalyst for progress, positioning the Akash Network as a leading force in the rapidly evolving landscape of DePin.

## 2024 Hackathons Timeline

- Early 2024 hackathon (Jan - March)
- Mid 2024 Hackathon (May - July)
- Late 2024 Hackathon (Sept - Nov.)

## Projected Outcomes

- 15 new Insiders to the Insider program
- 600 Total registrations 
- 75 Project submissions
- 3+ New open source tools
- 6 Sponsored Tracks (partnerships)
- 12 recorded workshops (video content)

## Individual Hackathon Timelines

Below is a timeline of activities in lead up to, during and after each proposed hackathon.

1. **1 month (Preparation)**
   - Develop marketing assets
   - Outreach and partnership with ecosystem players
   - Develop Hackathon tracks
   - Develop sponsored tracks
   - Define hackathon timeline
   - Define budget
   - Judge & mentor selection
   - Judging criteria

2. **2 weeks (Registration during preparation phase)**
   - Prepare final details of hackathon
   - Encourage signups

3. **1.5 months (Execution)**
   - Weekly mentor sessions
   - Dedicated website
   - Dedicated discord channel
   - Demo Day (top 3 pitches w/judges)
   - Review projects and select winners

4. **2 weeks (Review)**
   - Feedback forms
   - Announce winners
   - Post mortem with organizers & sponsors
   - Judges notes given to participants

## Hackathon Roles (Support)

- **Hackathon Community Manager:** Manages hacker events and hacker support
- **Hackathon Marketing Manager:** Leads hackathon judges confirmation and partner sponsored tracks
- **Event Manager:** Manages workshops hosted throughout the duration of the hackathon

Additional support not included in roles will be provided by members of the Overclock Labs team.

- **Judges x4**
- **Workshop Mentors x4**

## 2024 Hackathon Budget

Each hackathon will have increasing prize pools.

- **Early 2024 Hackathon - 50k USD in AKT**
- **Mid 2024 Hackathon - 75 USD in AKT**
- **Late 2024 Hackathon - 100k USD in AKT**

Dorahacks will cover the majority of prize pools, and the Akash community fund will cover up to the needed amount. Below are estimated amounts needed for each hackathon for prizes.

- **Early 2024 Hackathon - 10k USD in AKT**
- **Mid 2024 Hackathon - 30k USD in AKT**
- **Late 2024 Hackathon - 60k USD in AKT**

## Rewards

- **Dorahacks Prize allotment $150,000 (+/-) USD**
- **Akash Community Fund Prize allotment $100k USD**

## Hackathon Support Roles

- **$27,000**
- **Swag $4,500**
- **Paid Sponsor Tracks - Sponsor to cover cost**

## Individual Hackathon Budget

- **Hackathon Support Roles**
  - **Community Manager - $3,000**
  - **Event Manager - $3,000**
  - **Marketing Manager - $3,000**
  - **Total: $9,000**

## Rewards

- **Dorahacks Prize allotment $50,000**
- **Akash Community Fund Prize allotment $50,000**
- **Swag $1,500 USD**
- **Paid Sponsor Tracks - Sponsor to cover costs**
- **Total community pool request in AKT for 2024: $131,500 USD paid in AKT**

**Note:** Unused prize winnings will go back into the community pool

## Next Steps

This proposal will be open for public discourse until 12/22/23, at which time we will update with given community feedback and put on chain for a vote.

**Comment by bugrakaplan87** [comment]
You are cordially invited to join Areon Network’s hackathon ‘Code AREA’ as a collaborator! 

https://hackathon.areon.network/

This Web3 development competition features a $500K total prize and amazing growth opportunities for students and beginner developers!

The virtual competition will last for 6 weeks and end with a one-day offline event in Istanbul!

hackathon.areon.network
https://dorahacks.io/hackathon/code-area

**Comment by rodri-r** [comment]
Hey @CaptainSplince as we chatted in a call last week about these future hackathons, I just want to state here again that I really like these plans, the prizes and that there will be several hackathons throughout the year. It should draw more attention to the network, make competition a lot more interesting and can't wait to see what is submitted. 

As mentioned before too, I am interested in helping out with these hackathons. I was one of the organizers of the previous Akashathon and I am ready to help with any of the positions, as I helped with these tasks in the past.

**Reply by CaptainSplince** [reply]
Thanks @rodri-r. We're aiming to put this onchain for a vote in the new year!

**Comment by SophoclesSummer** [comment]
This proposal was approved on 1/23/2024 by a 98% majority. 
https://www.mintscan.io/akash/proposals/245
